include/dump_utils.hpp

```diff
@@ -35,25 +36,22 @@ inline void getValidDumpEntryForAttachment(
         // BMC type dump
         dumpType = "BMC";
         entriesPath = "/redfish/v1/Managers/bmc/LogServices/Dump/Entries/";
+        dumpId = entryID;
     }
     else if (crow::utility::readUrlSegments(
                  url, "redfish", "v1", "Systems", "system", "LogServices",
                  "Dump", "Entries", std::ref(entryID), "attachment"))
     {
-        // System type
-        dumpType = "System";
+        // System, Resource, and other types
         entriesPath = "/redfish/v1/Systems/system/LogServices/Dump/Entries/";
-    }
-    if (dumpType.empty() || entryID.empty())
-    {
-        redfish::messages::resourceNotFound(asyncResp->res, "Dump", entryID);
-        return;
-    }
 
-    std::string dumpId(entryID);
+        // System and Resource dump entries are currently being
```

Comment:
all these types system,resource,sbe,hwdump and host boot dumps are listed under /Systems/system/LogServices/Dump/Entries/
so please modify this note accordingly

Reply:
Comments are updated